### ISSUE TITLE
error: No matching distribution found for afvaldienst==0.6.78

### ISSUE THREAD

**Issue by Mariusthvdb** [xirixiz/Home-Assistant-Sensor-Afvalwijzer#60]
keep seeing this after having downloaded the latest version:

```
2020-04-30 00:27:11 ERROR (SyncWorker_10) [homeassistant.util.package] Unable to install package afvaldienst==0.6.78: ERROR: Could not find a version that satisfies the requirement afvaldienst==0.6.78 (from versions: 0.2.0, 0.3.0, 0.4.0, 0.6.0, 0.6.1, 0.6.2, 0.6.3, 0.6.4, 0.6.5, 0.6.6, 0.6.7, 0.6.8)
ERROR: No matching distribution found for afvaldienst==0.6.78
WARNING: You are using pip version 20.0.2; however, version 20.1 is available.
You should consider upgrading via the '/usr/local/bin/python3 -m pip install --upgrade pip' command.
```

and config fails:

<img width="592" alt="Schermafbeelding 2020-04-30 om 00 31 03" src="https://user-images.githubusercontent.com/33354141/80653198-ea9c2900-8a79-11ea-9a25-1b2b2a3bfba8.png">

I did accidentally edit the manifest file at first, incorrectly, but after that I corrected. No success though. I then dll'ed the new file again, cleared cache, refreshed etc etc. 

the error keeps popping up. What to do? Is the old value stored in .storage somewhere maybe?
even completely deleting the CC doesn't solve the issue.

And HA wont restart, to create a clean install without Afvalwijzer...
stuck in the middle of a non existing error.

**Comment by xirixiz** [comment]
Sure, because the release is 0.6.8 :).
https://pypi.org/project/afvaldienst/

**Comment by Mariusthvdb** [comment]
sure, you've probably misread what I posted.
After I had noticed I had made a typo, I corrected that in the manifest.

From that moment on, the error remained in the system no matter what I tried. Since it wasn't in the .storage folder, not in the yam files, I must have been in the system elsewhere. All I ask here is where that might be?

I've managed to get the system back up again, by taking out the CC And the yaml config for the integration. Simply putting these back, immediately lead to the config error as posted.
What I had to do was:
- take out the CC
- take out the yaml for the CC
- restart
- install CC
- restart
- re-install the yaml

seems a bit overcomplicated...
